packages/formik/src/index.tsx

```diff
@@ -1,3 +1,5 @@
+export * from './hooks/useFormikState';
+export * from './hooks/hooks';
```

Comment:
We are exporting `useOptimizedSelector`?

Reply:
In another PR I move useOptimizedSelector to a separate package. You should check that out before I modify this.

Also, I don't think I exported this but I'll double check.

Reply:
It is not exported.